pkgs/development/ocaml-modules/syslog/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, ocaml, findlib }:
+
+stdenv.mkDerivation rec {
+  pname = "ocaml-syslog";
```

Comment:
Please include the version of OCaml in the name.

Reply:
Ok, I've indeed found a few packages doing this and got inspiration from them.
But then it means users have to explicitly request a version such as "ocaml4.06.1-syslog"; isn't that laborious?

Reply:
Users barely use the name of a package; they more likely use its attribute path: `ocamlPackages.syslog` or, if a specific version is needed, `ocaml-ng.ocamlPackages_4_07.syslog`.

Reply:
I see